plugins/generator-1.18.1/forge-1.18.1/procedures/get_localized_text.java.ftl

```diff
@@ -0,0 +1 @@
+(net.minecraft.locale.Language.getInstance().getOrDefault(${input$key}))
```

Comment:
The previous method was cleaner, use 1.17.1 approach

Reply:
I didn't find `getString()` method on text components on MC 1.18, but I haven't checked yet whether it's present in 1.17 version.

Reply:
Check inheritance tree of TranslatableComponent, there is getString in net.minecraft.network.chat.getString, also in com.mojang.brigadier.Message, that TranslatableComponent both inherits.

Reply:
:heavy_check_mark: Fixed